src/ts/schemaDesigner/schemaDesigner.css

```diff
@@ -0,0 +1,174 @@
+.sd-toolbar-belt {
+    position: fixed;
+    bottom: 22px;
+    z-index: 8;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+  }
+  
+  .sd-toolbar {
+    height: 48px;
+    width: fit-content;
+    gap: 0.5rem;
+    display: flex;
+    background-color: var(--color-toolbar-bg);
+    color: var(--color-toolbar-text);
+    padding: 5px;
+    border-radius: 0.8rem;
+    vertical-align: middle;
+    justify-content: center;
+    align-items: center;
+    box-shadow: 0px 3px 8px rgba(0, 0, 0, 0.35), 0px 1px 3px rgba(0, 0, 0, 0.5),
+      inset 0px 0.5px 0px rgba(255, 255, 255, 0.08),
+      inset 0px 0px 0.5px rgba(255, 255, 255, 0.3);
+  }
+  
+  .sd-toolbar-button {
+    cursor: pointer;
+    width: 32px;
+    height: 32px;
+    background-size: 25px 25px;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 4px;
+    border-radius: 0.5rem;
+  }
+  
+  .sd-toolbar-button:hover {
+    background-color: var(--color-toolbar-button-hover);
+  }
+  
+  .sd-toolbar-divider {
+    height: 100%;
+    width: 1px;
+    background-color: var(--color-toolbar-divider);
+  }
+  
+  #graphContainer {
+    width: 100%;
+    height: 100%;
+    z-index: 1;
```

Comment:
you may consider creating variable for z-index values to be more expressive what you're trying to do with the different values?

for example, 

```CSS
root {
  /* Z-index hierarchy system */
  --z-base: 1;
  --z-graph: var(--z-base);
  --z-outline: calc(var(--z-base) + 4);
  --z-toolbar: calc(var(--z-base) + 7);
}

/* Then update the components to use these variables */
.sd-toolbar-belt {
  position: fixed;
  bottom: 22px;
  z-index: var(--z-toolbar);
  /* rest of styles... */
}

.sd-outline {
  position: fixed;
  right: 10px;
  top: 10px;
  z-index: var(--z-outline);
  /* rest of styles... */
}

#graphContainer {
  width: 100%;
  height: 100%;
  z-index: var(--z-graph);
  /* rest of styles... */
}
```